app/controllers/questions_controller.rb

```diff
@@ -31,6 +31,18 @@ def show
     authorize! :read, @question
     @responses = Response.all.where(question_id: params[:id])
     @response = Response.new
+    @title = ''
```

Comment:
consider the `truncate` helper to simplify this. http://api.rubyonrails.org/classes/String.html#method-i-truncate